doc/developer-guide/api/functions/TSContScheduleOnPool.en.rst

```diff
@@ -55,6 +55,69 @@ called and continuations that use them have the same restrictions. ``TS_THREAD_P
 are threads that exist to perform long or blocking actions, although sufficiently long operation can
 impact system performance by blocking other continuations on the threads.
 
+Example Scenarios
+=================
+
+Scenario 1 (no thread affinity info, different types of threads)
+----------------------------------------------------------------
+
+NO affine thread set, plugin is calling the API on thread "A", "A" is of type "ET_TASK", and plugin
```

Comment:
How about

When thread affinity is not set, a plugin calls the API on thread "A" (which is an "ET_TASK" type), and wants to schedule on an "ET_NET" type thread provided in "tp", the system would see there is no thread affinity information stored in "contp."